docs/dc-apps-performance-toolkit-user-guide-jira.md

```diff
@@ -84,12 +84,16 @@ The Data Center App Performance Toolkit framework is also set up for concurrency
 
 | Parameter | Recommended Value |
 | --------- | ----------------- |
-| Database instance class | [db.m5.large](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.DBInstanceClass.html#Concepts.DBInstanceClass.Summary) |
+| Database instance class | [db.m5.xlarge](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.DBInstanceClass.html#Concepts.DBInstanceClass.Summary) |
 | RDS Provisioned IOPS | 1000 |
 | Master (admin) password | Password1! |
 | Enable RDS Multi-AZ deployment | true |
 | Application user database password | Password1! |
 
+{{% note %}}
+The indexing process places high demands on the database. Weak database instance class can become a bottle neck, for long ping times or poor response from the database that can also lead to an error during the reindex process! 
```

Comment:
The Jira indexing is a database resource-intense process. Underprovisioned database instance could become a bottleneck which may lead to a slow index process or errors during the indexing. It's highly recommended to stick to a suggested database instance class.